<launch>

	<!-- Kinect driver -->
	<include file="$(find openni_launch)/launch/openni.launch" >
		<arg name="depth_registered" value="depth_registered_hr" />
	</include>

	<!-- Limit data rate -->
	<node name="rate_limiter_node" pkg="person_tracker" type="rate_limiter" args="1000" output="screen">
		<remap from="in_image"  to="/camera/depth_registered_hr/image_rect" />
		<remap from="out_image" to="/camera/depth_registered/image_rect" />

		<remap from="in_cloud"  to="/camera/depth_registered_hr/points" />
		<remap from="out_cloud" to="/camera/depth_registered/points" />
	</node>

</launch>
